bmc-log-collector/log-collector.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
+	"net/http"
+	"strconv"
+
+	"time"
+)
+
+type SystemEventLog struct {
+	Od_Id             string   `json:"@odata.id"`
+	Od_Type           string   `json:"@odata.type"`
+	Create            string   `json:"Created"`
+	Description       string   `json:"Description"`
+	EntryCode         string   `json:"EntryCode"`
+	EntryType         string   `json:"EntryType"`
+	GeneratorId       string   `json:"GeneratorId"`
+	Id                string   `json:"Id"`
+	Message           string   `json:"Message"`
+	MessageArgs       []string `json:"MessageArgs"`
+	OdCnt_MessageArgs int      `json:"MessageArgs@odata.count"`
+	MessageId         string   `json:"MessageId"`
+	Name              string   `json:"Name"`
+	SensorNumber      int      `json:"SensorNumber"`
+	SensorType        string   `json:"SensorType"`
+	Severity          string   `json:"Severity"`
+	Serial            string
+	NodeIP            string
+	BmcIP             string
+}
+
+type RedfishJsonSchema struct {
+	Name        string           `json:"Name"`
+	Count       int              `json:"Members@odata.count"`
+	Context     string           `json:"@odata.context"`
+	Id          string           `json:"@odata.id"`
+	Type        string           `json:"@odata.type"`
+	Description string           `json:"Descriptionta"`
+	Sel         []SystemEventLog `json:"Members"`
```

Comment:
The filed name is `Sel`, but the tag name is `Members`.
Is it corret?

Reply:
Please refer page 53 in PDF. It is the part of the common Redfish resource schema. Members is in collection of resorces depend that can vary on API path.
https://www.dmtf.org/sites/default/files/standards/documents/DSP2046_2023.3.pdf